torch/csrc/distributed/c10d/init.cpp

```diff
@@ -31,8 +30,10 @@
 #include <c10d/frontend.hpp>
 #include <c10d/logger.hpp>
 #include <c10d/reducer.hpp>
+#include <c10d/quantization/quantization_gpu.h>
 #include <torch/csrc/Exceptions.h>
 #include <torch/csrc/distributed/c10d/python_comm_hook.h>
+#include <torch/csrc/distributed/c10d/quantization/quantization.h>
```

Comment:
shouldn't this be the same as `<c10d/quantization/quantization_gpu.h>`? not sure why we have to include these two in two different ways, perhaps because the build_variables?